src/net/stackerdb/tests/db.rs

```diff
@@ -0,0 +1,580 @@
+// Copyright (C) 2013-2020 Blockstack PBC, a public benefit corporation
+// Copyright (C) 2020-2023 Stacks Open Internet Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+use std::fs;
+
+use crate::net::stackerdb::{
+    db::stackerdb_table, db::ChunkValidation, ChunkMetadata, StackerDB, StackerDBConfig,
+};
+
+use crate::net::ContractId;
+use crate::net::Error as net_error;
+use crate::net::StackerDBChunkData;
+
+use clarity::vm::ContractName;
+use stacks_common::types::chainstate::ConsensusHash;
+use stacks_common::types::chainstate::StacksAddress;
+
+use stacks_common::util::hash::Hash160;
+use stacks_common::util::hash::Sha512Trunc256Sum;
+use stacks_common::util::secp256k1::MessageSignature;
+
+use stacks_common::address::{
+    AddressHashMode, C32_ADDRESS_VERSION_MAINNET_MULTISIG, C32_ADDRESS_VERSION_MAINNET_SINGLESIG,
+};
+use stacks_common::types::chainstate::{StacksPrivateKey, StacksPublicKey};
+
+/// Test that we can instantiate a stacker DB instance
+#[test]
+fn test_stackerdb_connect() {
+    let path = "/tmp/test_stackerdb_connect.sqlite";
```

Comment:
Can you prefix these paths with `/tmp/stacks-node-tests/stackerdb/`? It makes cleaning up `/tmp` much easier after running the test suite.

Reply:
Sure